local_build_dotnet_core.sh

```diff
@@ -0,0 +1,293 @@
+#!/usr/bin/env bash
```

Comment:
Please remove this script.

Reply:
Agree. I will soon remove it. As an alternative, could I put these information into a *.md file as I find it useful for newbie like myself to learn how to do local build?

Reply:
Sorry. I wouldn't list all the reasons in details, but I'd just say this repo is not a place to create a page for "Docker 101: tutorial". I suggest you write your own blog (like [this one](https://medium.com/better-programming/how-to-harden-your-containers-with-distroless-docker-images-c2abd7c71fdb)).

Reply:
Thanks for the feedback. I will soon remove it.